.gitignore

```diff
@@ -1,8 +1,15 @@
+*.iml
 .gradle
 /local.properties
-/.idea/workspace.xml
+/.idea/caches
 /.idea/libraries
-.idea
+/.idea/modules.xml
```

Comment:
Delete all .idea subfolder entries and include the whole .idea folder.

Reply:
Done